model/member_invite.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+type MemberInvite struct {
+	Emails   []string `json:"emails"`
+	Channels []string `json:"channels,omitempty"`
```

Comment:
Should we rename this to `ChannelIds` so that it's clear that is not an array of channels?